adapters/akka-http/src/main/scala/caliban/AkkaHttpAdapter.scala

```diff
@@ -186,11 +186,18 @@ trait AkkaHttpAdapter {
                   .onExit {
                     case Failure(cause) if !cause.interrupted =>
                       IO.fromFuture(_ =>
-                          sendTo.offer(TextMessage(json.encodeWSError(messageId, cause.squash.toString)))
+                          sendTo
+                            .offer(TextMessage(json.encodeWSError(messageId, cause.squash.toString)))
+                            .recover { case _ => }
```

Comment:
Shall we just use `.ignore` instead of `orDie` and `recover`?

Reply:
Thanks for the tip, I have updated the PR.  Every time I do a Caliban PR, I always end up getting free ZIO lessons 😄